runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -201,24 +202,46 @@ public Query query(@Nonnull final String namespace) {
     /**
      * Subscribe to a specific a table in a namespace or the entire namespace.
      * Objects returned will honor transactional boundaries
-     * @param streamListener - callback context.
-     * @param namespace - the CorfuStore namespace to subscribe to.
-     * @param tablesOfInterest - If specified, only updates from these tables will be returned.
-     * @param timestamp - If specified, all stream updates from this timestamp will be returned
-     *                  - if null, only future updates will be returned.
+     *
+     * @param streamListener   callback context
+     * @param namespace        the CorfuStore namespace to subscribe to
+     * @param tablesOfInterest only updates from these tables of interest will be returned
+     * @param timestamp        if specified, all stream updates from this timestamp will be returned
+     *                         if null, only future updates will be returned
      */
+    @Deprecated
     public <K extends Message, V extends Message, M extends Message>
     void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace,
                    @Nonnull List<TableSchema<K, V, M>> tablesOfInterest,
                    @Nullable Timestamp timestamp) {
         runtime.getTableRegistry().getStreamManager()
                 .subscribe(streamListener, namespace, tablesOfInterest,
-                (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence());
+                        (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence());
+    }
+
+    /**
+     * Subscribe to a specific a table in a namespace or the entire namespace.
```

Comment:
typo: Subscribe to a specific table in a namespace or all tables in the namespace

Reply:
Done.